### DISCUSSION TITLE
Does mysqldump count as usage?

### DISCUSSION THREAD

**Discussion by mezuky04** [planetscale/discussion#518] [Q&A] [answered]
Hello. I would like to back up my PlanetScale database using `mysqldump`. Does this count as usage (e.g. rows reads)? What about when a dump is imported? Basically, I would need to have a copy of the data in another location, not managed or related to PlanetScale. Do you recommend something better than `mysqldump`? Thanks

**Comment by orware** [comment, marked as answer]
Hello @mezuky04,

Yes, creating a manual backup would count against your rows read usage for the current month. And similarly, importing that data back in would also count towards your row writes as well.

Generally, we don't recommend using `mysqldump` due to some of the [following issues](https://vitess.io/docs/16.0/user-guides/configuration-basic/exporting-data/#mysqldump) described in the Vitess documentation, but even if you are making use of our `pscale database dump` [command](https://planetscale.com/docs/reference/database) the manual backups would count towards your row reads.

If you are on one of our Scaler Pro plans however that would be less of a concern since those include unlimited row reads/writes so that would be one potential way to avoid the worry, but on our Hobby/Scaler plans it would still be something you would want to more carefully consider.

I hope this information helps to answer the questions you had!
